getting-started/windows-calico/limitations.md

```diff
@@ -71,6 +71,11 @@ Felix must reprogram the HNS ACL policy attached to the pod. This reprogramming
 
 On Windows nodes, kube-proxy unconditionally applies source NAT to traffic from local pods to service ClusterIPs. This means that, at the destination pod, where policy is applied, the traffic appears to come from the source host rather than the source pod. In turn, this means that a network policy with a source selector matching the source pod will not match the expected traffic.
 
+**Managed EKS**
+
+Currently, managed EKS Linux nodes ship with kube-proxy WinDSR disabled. However, WinDSR is required for network policy to be enforced for service ClusterIPs.
```

Comment:
`managed EKS Windows nodes`

I suggest removing `For production EKS usage, we recommend using self-managed EKS Linux nodes.`